docs/standard/library-guidance/breaking-changes.md

```diff
@@ -88,6 +88,10 @@ public class Document
 }
 ```
 
+**✔️ CONSIDER** keeping types and methods with the <xref:System.ObsoleteAttribute> indefinitely in low and middle-level libraries.
+
+> Removing an obsolete type or method is a binary breaking change. Not removing them can help avoid the worst-case scenarios mentioned above.
```

Comment:
Overall I like the advice. I think it does depend on the situation. If the obsolete API is rarely used, or keeping it creates a lot of technical debt then I think removing it after some time is a good idea.

```suggestion
> Removing APIs is a binary breaking change. Considering keeping the obsolete types and methods if maintaining them doesn't add lot of technical debt to your library. Not removing types and methods can help avoid the worst-case scenarios mentioned above.
```